android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -379,6 +380,7 @@ public void setSource(WebView view, @Nullable ReadableMap source) {
           }
         }
         view.loadUrl(url, headerMap);
+        view.requestFocus();
```

Comment:
Remove this so focus only happens when user calls the function

Reply:
Forget about this line.

Reply:
https://github.com/react-native-community/react-native-webview/pull/567/commits/5c31cf3b12ba94569b17bd2fdabb2d54518d8b86